### ISSUE TITLE
Sometimes deletes unwanted double transactions

### ISSUE THREAD

**Issue by OscarvanDijk** [stant/mdcsvimporter2015#6]
I import a 205 row file to the Import account. The import account is empty. Sometimes the status reports 205 rows found, found 2 double 203 rows imported. 
I would like to know what the classification for the double transaction is. 

Br. Oscar

**Comment by stant** [comment]
Hello Oscar,
I use original transaction values, not what you may have edited them to.
I concatenate:
DateInitiated:Amount:PayeeName[Description]:CheckNumber [without leading zeros]:Memo
Stan
-------- Original message --------From: OscarvanDijk notifications@github.com Date: 4/30/2016  10:34 AM  (GMT-05:00) To: stant/mdcsvimporter2015 mdcsvimporter2015@noreply.github.com Subject: [stant/mdcsvimporter2015] Sometimes deletes unwanted double
  transactions (#6) 
I import a 205 row file to the Import account. The import account is empty. Sometimes the status reports 205 rows found, found 2 double 203 rows imported. 

I would like to know what the classification for the double transaction is. 

Br. Oscar

**Comment by OscarvanDijk** [comment]
Hi Stan,

Thanks, I have found the reason for the double in my CSV file. My bank does not use the CheckNumber, Some of the entries had everything the same including the amount and the Payee. I have now changed my CSV file a little. This has solved the problem. 
Oscar

**Comment by stant** [comment]
Is that valid, what the bank gives you?

What do you change in your file and will you have to do that all the time
to import it?

Stan

**Comment by OscarvanDijk** [comment]
The data from the bank is valid, I have checked this. The issue is that I have a payment split over 2 separate accounts. The split is 50/50 so these showed up as doubles.
The strange thing is that some of the entries were OK while others were marked as double and not processed. 
Since I have to modify the CSV file anyway, I have now added the account number to the Payee. This solved the problem. 
And yes, I need to do this all the time. It might not be that big issue in the future but I imported 4 months of data this time. 

Oscar
